cmd/podman/containers/stats.go

```diff
@@ -166,7 +168,7 @@ func outputStats(reports []define.ContainerStats) error {
 	if report.IsJSON(statsOptions.Format) {
 		return outputJSON(stats)
 	}
-	format := "{{.ID}}\t{{.Name}}\t{{.CPUPerc}}\t{{.MemUsage}}\t{{.MemPerc}}\t{{.NetIO}}\t{{.BlockIO}}\t{{.PIDS}}\n"
+	format := "{{.ID}}\t{{.Name}}\t{{.UpTime}}\t{{.CPUPerc}}\t{{.AVGCPU}}\t{{.MemUsage}}\t{{.MemPerc}}\t{{.NetIO}}\t{{.BlockIO}}\t{{.PIDS}}\n"
```

Comment:
Changing the default format would break docker compat. Not sure if this is desired. @mheon @rhatdan WDYT?

Reply:
I am not sure I would classify this as a breaking change to compat, but it would probbaly be best if we added the new fields to the end of the output, that way, if someone built a script for docker that expected the second field to be CPUPerc, we would not break it.

Reply:
Do you want me to switch around the ordering? I feel like that would make the output look strange though to have CPU data all scattered throughout.

Reply:
Unfortunately, I think we need to @cdoern

Reply:
Maybe add an `--alpha` flag which would then show the stats in alphabetical order left to right?  Or if you really want to go off the rails, make it all selectable by the caller.

Reply:
@TomSweeneyRedHat working on an alpha flag right now, should it simply just be an if/else around `format:=...` with two different options?

Reply:
@TomSweeneyRedHat I don't see the value for an extra alpha flag, the user can always change the output format with `--format`

Reply:
Tend to agree - I don't see a strong case for this until and unless we get a user request to add it.

Reply:
Ok, so we are not going to move the new data to the end of stats?  And just tell users if they want a specific format to use --format.  I guess I am good with this.  Make the default look good for the user.

Reply:
@Luap99 The value in my eye is just typing `--alpha` rather than working it out in the `--format` option.  YMMV.  I think it would be a good addition, but not a biggy.